UKHomeOffice/nmsw-ui
# NMSW-46 Add feedback URL to link in Phase Banner & a feedback banner to bottom of page

# Ticket

NMSW-46

----

## AC

Add the feedback link URL to the phase banner and build a feedback component to be shown at the bottom of the page (see prototype)

----

## To test

- Run app
- Go to localhost:3000
- Click on the link to feedback in the phase banner
- > You should be taken to the feedback survey page
- Go back to the application
- You should see a feedback banner at the bottom to the page
- > Compare this banner to the banner in the prototype
- Click on the short survey link
- > It should open in a new tab

----

## Notes
